freetextresponse/private/view.html

```diff
@@ -1,24 +1,29 @@
-<div class="freetextresponse">
+<div class="freetextresponse xmodule_display xmodule_CapaModule">
     <h2 class="problem-header">{self.display_name}</h2>
     <div class="problem-progress">{problem_progress}</div>
-    <p>{self.prompt}</p>
-    <div class="user-input {indicator_class}">
-        <textarea class="student_answer" style="height:150px" rows="20" cols="70">{self.student_answer}</textarea>
-        <span class="status {indicator_visibility_class}" aria-describedby="student_answer"></span>
-    </div>
-    <div class="submission-received">
-        {submitted_message}
-    </div>
-    <div class="word-count-error">
-        {word_count_message}
-    </div>
-    <div class="action">
-        <span class="{submit_class}">
-            <button class="check Submit" data-checking="Checking..." data-value="Submit" aria-disabled="false">
+    <div class="problem">
+        <p>{self.prompt}</p>
+        <div class="capa_inputtype textline {indicator_class}">
+            <div class="{indicator_class}">
+                <textarea class="textarea student_answer" style="height:150px; box-sizing:border-box; border-radius:3px; border:2px solid #e4e4e4; min-width:160px;" rows="20" cols="70">{self.student_answer}</textarea>
```

Comment:
I know this predates you, but we should move this to SASS, instead of styling inline.